content/posts/newsletter-015/index.md

```diff
@@ -60,6 +60,21 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Antorum Online]
```

Comment:
You only need the `[]` around the title if you want it to be a link - in [the example](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md#style), note that there's a corresponding URL for `[Project]` at the end. Otherwise you just get [literal square brackets in the text](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/bc83679d45c367483cc6985d2c813252478b280d/content/posts/newsletter-015/index.md#antorum-online).

It's also preferred to do that style of link for the links in your content, rather than having them inline, but I think for a link list like the one you've got, having them inline is probably okay.